From 88fee5c6eb2271d3de6b9878cd29a0494999aa18 Mon Sep 17 00:00:00 2001
From: "Arnout Vandecappelle (Essensium/Mind)" <arnout@mind.be>
Date: Sun, 20 Nov 2016 23:46:45 +0100
Subject: [PATCH] Use correct variables for GnuTLS detection

At least with recent CMake, it seems the variables created by
find_package (GnuTLS) are called PC_GNUTLS_INCLUDE_DIRS and
PC_GNUTLS_LIBRARIES.

Signed-off-by: Arnout Vandecappelle (Essensium/Mind) <arnout@mind.be>
---
 CMakeLists.txt | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index d8125d9..ccf827b 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -110,8 +110,8 @@ message ("-- Looking for GnuTLS")
 find_package (GnuTLS REQUIRED)
 if (GNUTLS_FOUND)
   set (HAVE_LIBGNUTLS true)
-  set (TASKD_INCLUDE_DIRS ${TASKD_INCLUDE_DIRS} ${GNUTLS_INCLUDE_DIR})
-  set (TASKD_LIBRARIES    ${TASKD_LIBRARIES}    ${GNUTLS_LIBRARIES})
+  set (TASKD_INCLUDE_DIRS ${TASKD_INCLUDE_DIRS} ${PC_GNUTLS_INCLUDE_DIRS})
+  set (TASKD_LIBRARIES    ${TASKD_LIBRARIES}    ${PC_GNUTLS_LIBRARIES})
 endif (GNUTLS_FOUND)
 
 message ("-- Looking for libuuid")
-- 
2.10.2

